lib-multisrc/colamanga/build.gradle.kts

```diff
@@ -2,8 +2,11 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 9
+baseVersionCode = 10
 
 dependencies {
     api(project(":lib:synchrony"))
+    implementation(project(":lib:dataimage"))
+    implementation(project(":lib:randomua"))
+    implementation(project(":lib:i18n"))
```

Comment:
remove i18n dependency as well when moving to `src/zh/onemanhua` as only Chinese strings are relevant, so they should be inlined